site/content/docs/include/common-svc-fields.md

```diff
@@ -0,0 +1,180 @@
+<div class="separator"></div>
+
+<a id="entrypoint" href="#entrypoint" class="field">`entrypoint`</a> <span class="type">String or Array of Strings</span>
+Override the default entrypoint in the image.
+```yaml
+# String version.
+entrypoint: "/bin/entrypoint --p1 --p2"
+# Alteratively, as an array of strings.
+entrypoint: ["/bin/entrypoint", "--p1", "--p2"]
+```
+
+<div class="separator"></div>
+
+<a id="command" href="#command" class="field">`command`</a> <span class="type">String or Array of Strings</span>
+Override the default command in the image.
+
+```yaml
+# String version.
+command: ps au
+# Alteratively, as an array of strings.
+command: ["ps", "au"]
+```
+
+<div class="separator"></div>
+
+<a id="cpu" href="#cpu" class="field">`cpu`</a> <span class="type">Integer</span>
+Number of CPU units for the task. See the [Amazon ECS docs](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html) for valid CPU values.
+
+<div class="separator"></div>
+
+<a id="memory" href="#memory" class="field">`memory`</a> <span class="type">Integer</span>
+Amount of memory in MiB used by the task. See the [Amazon ECS docs](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html) for valid memory values.
+
+<div class="separator"></div>
+
+<a id="count" href="#count" class="field">`count`</a> <span class="type">Integer or Map</span>
+If you specify a number:
+```yaml
+count: 5
+```
+The service will set the desired count to 5 and maintain 5 tasks in your service.
+
+If you want to use Fargate Spot capacity to run your services, you can specify a number under the `spot` subfield:
+```yaml
+count:
+  spot: 5
+```
+
+<span class="parent-field">count.</span><a id="count-spot" href="#count-spot" class="field">`spot`</a> <span class="type">Integer</span>
+
+Alternatively, you can specify a map for setting up autoscaling:
+```yaml
+count:
+  range: 1-10
+  cpu_percentage: 70
+  memory_percentage: 80
+  requests: 10000
+  response_time: 2s
+```
+
+<span class="parent-field">count.</span><a id="count-range" href="#count-range" class="field">`range`</a> <span class="type">String or Map</span>
```

Comment:
Should this be `Integer or Map`? 
I believe ranges of ints like `1-5` are considered ints?

Reply:
Sure, but the range as a whole is considered a string: https://github.com/aws/copilot-cli/blob/mainline/internal/pkg/manifest/svc.go#L75

Reply:
Ah, gotcha. I had checked with `reflect.TypeOf` but didn't see that we were handling these as strings. Thank you!